src/test/test_group.c

```diff
@@ -270,6 +274,55 @@ void test_shutdown_handler(crt_rpc_t *rpc_req)
 	printf("tier1 test_srver set shutdown flag.\n");
 }
 
+static struct crt_proto_rpc_format my_proto_rpc_fmt_test_group1[] = {
+	{
+		.prf_flags	= 0,
+		.prf_req_fmt	= &CQF_test_ping_check,
+		.prf_hdlr	= test_checkin_handler,
+		.prf_co_ops	= NULL,
+	}, {
+		.prf_flags	= CRT_RPC_FEAT_NO_REPLY,
+		.prf_req_fmt	= NULL,
+		.prf_hdlr	= test_shutdown_handler,
+		.prf_co_ops	= NULL,
+	}, {
+		.prf_flags	= CRT_RPC_FEAT_NO_TIMEOUT,
+		.prf_req_fmt	= &CQF_crt_test_ping_delay,
+		.prf_hdlr	= test_ping_delay_handler,
```

Comment:
indentation, should align at first letter

Reply:
Could'nt understand this one. The indentation is set to 8 chars. Its same in all other files and looks similar.